CHANGELOG.md

```diff
@@ -6,8 +6,10 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added option to pass custom headers to 'AWSV4SignerAsyncAuth' ([863](https://github.com/opensearch-project/opensearch-py/pull/863))
 - Added sync and async sample that uses `search_after` parameter ([859](https://github.com/opensearch-project/opensearch-py/pull/859))
 ### Updated APIs
+- Updated opensearch-py APIs to reflect [opensearch-api-specification@a2afe56](https://github.com/opensearch-project/opensearch-api-specification/commit/a2afe567254c42aee70b74a8d7d7228c982b56db)
```

Comment:
This seems unrelated, remove.

Reply:
was confused because it auto-popped somehow